src/app/education-page/education-page.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="main">
+    <p class="title">E D U C A T I O N</p>
```

Comment:
Ogólnie jak się pisze zmodyfikowane napisy (tutaj: uppercase, letter-spacing) to używa się klas cssowych i/lub JSa do tego. Zmień klasy dla tego p w taki sposób, żeby napis w kodzie wyglądał tak: "Education" a nie tak: "E D U C A T I O N", ale na stronie dalej było "E D U C A T I O N"